core/src/main/scala/com/gu/AppIdentity.scala

```diff
@@ -102,23 +103,30 @@ object AppIdentity {
     } yield DevIdentity(defaultAppName)
   }
 
-  def region: String = {
-    lazy val ec2Region = safeAwsOperation("Failed to identify the regionName of the instance") {
+  def region: Try[String] = {
+    lazy val ec2Region = safeAwsOperation("Running in DEV? Don't call this - Timed out trying to identify the regionName of the instance") {
         EC2MetadataUtils.getEC2InstanceRegion
       }
     lazy val lambdaRegion = Option(System.getenv("AWS_DEFAULT_REGION"))
-    lambdaRegion orElse ec2Region getOrElse "eu-west-1"
+    lambdaRegion.map(Success(_)).getOrElse(ec2Region)
   }
 
+  /**
+   * Don't call this when running in DEV, as it will spend around 7 seconds timing out outside of EC2
+   * @param defaultAppName
+   * @param credentials
+   * @return
+   */
   def whoAmI(
-              defaultAppName: String,
-              credentials: => AwsCredentialsProvider = DefaultCredentialsProvider.create()
-            ): AppIdentity = {
+    defaultAppName: String,
+    credentials: => AwsCredentialsProvider
+  ): Try[AppIdentity] = {
```

Comment:
now this will return a Try, which would contain a suitable message if it's run in DEV, to draw attention to the fact it had to time out before it realised it's in DEV